src/components/SideBar/SideBarItem.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, Icon, Text, Flex } from "@chakra-ui/react";
+
+const SideBarItem = (props: any) => {
+  return (
+    <a data-testid={props.item_text}>
```

Comment:
The item of this sidebar should be used to open different sidebar content, the a tag is generally used for navigation, such as <a href="/home" ></a>